### DISCUSSION TITLE
Pattern restriction in asset file names ?

### DISCUSSION THREAD

**Discussion by mistur** [requarks/wiki#6436] [Help / Questions]
Hello,

I have a wikijs based on image: ghcr.io/requarks/wiki:2

I have uploaded multiple files to the folder /public/files, it works for some files but not for all :

Assets access for Guest is : 

```
read:assets read:pages | Path Starts With... | /public
```

it works for : 
```
[admin.conf](/public/files/admin.conf)
[example.yaml](/public/files/example.yaml)
[k8s.conf](/public/files/k8s.conf)
```
but not for : 
```
[file.txt](/public/files/file.txt)
[file](/public/files/file)
[config.txt](/public/files/config.txt)
[config](/public/files/config)
```

is there any pattern filtering on files names ?

thanks for your help,

Best regards,

Yoann

**Comment by NGPixel** [comment]
Make sure you don't have other rules interfering with this one.

**Comment by mistur** [comment]
Hello, 
This is a new installation, we only have 2 groups (Administrators and Guests).

We only have 2 rules for Guests : 

![Screenshot 2023-05-19 at 09-06-22 Admin](https://github.com/requarks/wiki/assets/15873680/a887499f-a35c-46dc-994b-4903f2b7dcba)

Best regards,

Yoann
